app/login.go

```diff
@@ -113,6 +114,13 @@ func (a *App) DoLogin(w http.ResponseWriter, r *http.Request, user *model.User,
 		secure = true
 	}
 
+	domain := ""
+	if *a.Config().ServiceSettings.AllowCookiesForSubdomains {
+		if siteURL, err := url.Parse(*a.Config().ServiceSettings.SiteURL); err == nil {
+			domain = siteURL.Hostname()
```

Comment:
This config setting makes me very nervous. If we had the admin specify a single subdomain, could we just set multiple cookies instead?

Reply:
I guess we can't just set cookies for a single subdomain like that, can we... (I'm a bit fuzzy on the rules.)

Reply:
Ya. It makes me nervous too. I am not submitting a PR to add it to the system console. Really the only situation I see it used in is this websocket bypass.

If anyone can think of an alternate way to go about this, that would be great.